api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -128,4 +134,9 @@ public ParticipantCohortStatus findParticipantCohortStatus(Long cohortReviewId,
     public Slice<ParticipantCohortStatus> findParticipantCohortStatuses(Long cohortReviewId, PageRequest pageRequest) {
         return participantCohortStatusDao.findByParticipantKey_CohortReviewId(cohortReviewId, pageRequest);
     }
+
+    @Override
+    public Map<String, Map<Long, String>> findGenderRaceEthnicityFromConcept() {
```

Comment:
IMHO you could just inject Provider<GenderRaceEthnicityConcept> into the places calling this method. But if you do want to keep this method, I'd have it just return GenderRaceEthnicityConcept (which is easier to understand than returning this map.)

Reply:
Makes sense. Done